github/github.go

```diff
@@ -1259,7 +1259,8 @@ func CheckResponse(r *http.Response) error {
 		}
 	case r.StatusCode == http.StatusForbidden &&
 		(strings.HasSuffix(errorResponse.DocumentationURL, "#abuse-rate-limits") ||
-			strings.HasSuffix(errorResponse.DocumentationURL, "#secondary-rate-limits")):
```

Comment:
Would checking here also for `secondary rate limit` in the `message` be more resilient to future documentation updates?

Reply:
IMO, either or should be fine.